addons/tacticalladder/data/model.cfg

```diff
@@ -26,125 +26,118 @@ class CfgSkeletons {
+                                                    "Face_chopLeft","Face_Jawbone",
+                                                    "Face_LipLowerMiddle","Face_Jawbone",
+                                                    "Face_LipLowerLeft","Face_Jawbone",
+                                                    "Face_LipLowerRight","Face_Jawbone",
+                                                    "Face_Chin","Face_Jawbone",
+                                                    "Face_Tongue","Face_Jawbone",
+                                                "Face_CornerRight","Face_Hub",
+                                                    "Face_CheekSideRight","Face_CornerRight",
+                                                "Face_CornerLeft","Face_Hub",
+                                                    "Face_CheekSideLeft","Face_CornerLeft",
+                                                "Face_CheekFrontRight","Face_Hub",
+                                                "Face_CheekFrontLeft","Face_Hub",
+                                                "Face_CheekUpperRight","Face_Hub",
+                                                "Face_CheekUpperLeft","Face_Hub",
+                                                "Face_LipUpperMiddle","Face_Hub",
+                                                "Face_LipUpperRight","Face_Hub",
+                                                "Face_LipUpperLeft","Face_Hub",
+                                                "Face_NostrilRight","Face_Hub",
+                                                "Face_NostrilLeft","Face_Hub",
+                                                "Face_Forehead","Face_Hub",
+                                                    "Face_BrowFrontRight","Face_Forehead",
+                                                    "Face_BrowFrontLeft","Face_Forehead",
+                                                    "Face_BrowMiddle","Face_Forehead",
+                                                    "Face_BrowSideRight","Face_Forehead",
+                                                    "Face_BrowSideLeft","Face_Forehead",
+                                                "Face_Eyelids","Face_Hub",
+                                                "Face_EyelidUpperRight","Face_Hub",
+                                                "Face_EyelidUpperLeft","Face_Hub",
+                                                "Face_EyelidLowerRight","Face_Hub",
+                                                "Face_EyelidLowerLeft","Face_Hub",
+                                                "EyeLeft","Face_Hub",
+                                                "EyeRight","Face_Hub"
+        };// end of skeleton array
         // location of pivot points (local axes) for hierarchical animation
-        pivotsModel="A3\anims_f\data\skeleton\SkeletonPivots.p3d";
+        pivotsModel="Samples_F\TemplateRTM\ManSkeleton_Pivots.p3d";
```

Comment:
Changing pivots model. The old one was correct. Samples_F doesn't exist in normal play

Reply:
That is also wrong in the fastroping one and https://github.com/acemod/ACE3/pull/6773

Though afaik that entry isn't even used in-game, it's only needed for creation of rtm animations.